cheatsheets/Cross-Site_Request_Forgery_Prevention_Cheat_Sheet.md

```diff
@@ -127,7 +127,7 @@ We recommend researching if the framework you are using has an option to achieve
 
 Most developers tend to ignore CSRF vulnerability on login forms as they assume that CSRF would not be applicable on login forms because user is not authenticated at that stage. That assumption is false. CSRF vulnerability can still occur on login forms where the user is not authenticated, but the impact/risk view for it is quite different from the impact/risk view of a general CSRF vulnerability (when a user is authenticated).
 
-With a CSRF vulnerability on login form, an attacker can make a victim login as them and learn behavior from their searches. For more information about login CSRF and other risks, see section 3 of [this](https://seclab.stanford.edu/websec/csrf/csrf.pdf) paper.
+With a CSRF vulnerability on login form, an attacker can make a victim login with an account they control. The attacker can than learn behavior from their interaction with the site with features such as search/activity/navigation history. The victim might also end up trusting the site to perform operations such as providing highly confidential information such as payment details assuming that they are logged in to their account. For more information about login CSRF and other risks, see section 3 of [this](https://seclab.stanford.edu/websec/csrf/csrf.pdf) paper.
```

Comment:
The PDF referenced is very old (2008 according to PDF properties and this [link](http://www.adambarth.com/papers/2008/barth-jackson-mitchell-b.pdf)): One of the example given is related to *iGoogle* and this service [is not active anymore](https://en.wikipedia.org/wiki/IGoogle).

A more recent and up-to-date source regarding *Login CSRF* must be added with a description of the attack flow in order to allow the dev team to better understand the security issue.

The content added is a kind of resume of the content of the section 3 of the PDF but the explanation of the PDF allow to better understand the different scenario in which *Login CSRF* can be applied, so for me, the actual section invite the reader to read the section 3 of the PDF and this one contains well explanation.

@ThunderSon @mackowski What do you think?

Reply:
Sounds good to me. "For more information on login CSRF and it's associated risks" might be a better wording

Reply:
I agree with @righettod. Basically PDF describes login CSRF very well. 
I think that if there will be a problems with finding newer reference we can put here better and wider description of Login CSRF, based on that PDF. And reference that this description is based on this paper. 

So I think that we should either not try to explain it here but only invite the reader to read the section 3 of the PDF (or newer reference) OR try to rewrite it in a way that reader will be able to understand it well.